doc/classes/Line2D.xml

```diff
@@ -17,7 +17,7 @@
 			<argument index="0" name="position" type="Vector2">
 			</argument>
 			<description>
-				Add a point at the x/y position in the supplied [Vector2]
+				Add a point at the x/y position in the supplied [Vector2]. The point will be added at the end of the line.
```

Comment:
Don't use passive voice. See the [writing guidelines](http://docs.godotengine.org/en/latest/community/contributing/docs_writing_guidelines.html). The original description (the first sentence) also needs a fix. Example: `Add a point at the [code]position[/code]. Appends the point at the end of the line.`
_Note: I'm not sure of the description. Is there a limit to the number of points you can add?_

Reply:
Just noticed this one, I thought x/y meant 'x divided by y' by skimming through. Might be better to use (x, y) or just 'x and y'.